src/convert.cpp

```diff
@@ -113,26 +121,37 @@ ddwaf_object* to_ddwaf_object(
   }
```

Comment:
Can't select it in a comment but the block in line 114 should be using to_ddwaf_string too